src/utils/helpers.ts

```diff
@@ -87,3 +87,20 @@ export const appendEthAPIKey = (
 export function escapeRegExp(string: string): string {
   return string.replace(/[.*+?^${}()|[\]\\]/g, '\\$&'); // $& means the whole matched string
 }
+
+export async function retryPromise(

```

Comment:
Should we add a `maxTries` arg?

Reply:
https://github.com/DXgovernance/dxvote/pull/795/commits/97e931ccfee1013cce4e716dd240c662c533eb18